src/CppUTest/CommandLineTestRunner.cpp

```diff
@@ -113,8 +113,9 @@ int CommandLineTestRunner::runAllTests()
 {
     initializeTestRun();
     int loopCount = 0;
-    int failureCount = 0;
-    int repeat_ = arguments_->getRepeatCount();
+    int failedTestCount = 0;
+    int failedExecutionCount = 0;
+    int repeatCount = arguments_->getRepeatCount();
```

Comment:
This renaming from `repeat_` can be considered style preference, but aligning naming makes sense in my opinion. Also in this class names ending with underscore `_` are member variables, while local variables do not end with underscores.
If desired, I can roll back, but IMO the result will look odd with 3 variables using one convention and one using different convention.

Reply:
Makes sense, thanks!